Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "raft-engine"
 version = "0.4.2"
 authors = ["The TiKV Project Developers"]
 edition = "2018"
-rust-version = "1.75.0"
+rust-version = "1.85.0"
```

Comment:
Out of curiosity, does `fs2::FileExt` require the new rust version?

Reply:
No, it was not because of fs2::FileExt. It was because raft-engine as a lib does not have Cargo.lock, and one of the dependency`half` (required by citron) that requires 1.85+.
This is fine, since TiKV master branch uses 1.85+ nightly